apis/apim/v1alpha1/ratelimitpolicy_types.go

```diff
@@ -183,6 +195,35 @@ type RateLimitPolicyStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
+
+	// GatewaysRateLimits shows the rate limit configuration applied by policies at the gateway level
+	// this field is only meant for rate limit policies targeting a route
+	// +optional
+	GatewaysRateLimits []GatewayRateLimits `json:"gatewaysRateLimits,omitempty"`
```

Comment:
This here means a Route RLP could have multiple GW RLP modifying it?

Reply:
Yes, the HTTPRoute can have multiple parentrefs https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1alpha2.CommonRouteSpec